Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_6_27.md

```diff
@@ -0,0 +1,16 @@
+
+#### Incident Fields
+
+Updated the following fields to no longer be hidden.
+- **ASM - Service Detection**
+- **ASM - Service Owner Unranked Raw**
+- **ASM - Data Collection**
+
+#### Playbooks
+
+##### Cortex ASM - Remediation Path Rules
+
+- Updated the playbook to work on playbook re-run.
+##### Cortex ASM - ASM Alert
+
+- Updated the playbook to work on playbook re-run by initializing all fields in beginning.
```

Comment:
```suggestion
- Updated the playbook to work on playbook rerun by initializing all fields in the beginning.
```

Are you initializing all fields or clearing the values from all fields?

Reply:
could be effectively doing both depending if it is on first playbook run or not.  Mostly doing the 2nd (clearing values).